src/polkadot-test.js

```diff
@@ -1,97 +1,130 @@
-import Keyring from '@polkadot/keyring';
-import { mnemonicGenerate, cryptoWaitReady } from '@polkadot/util-crypto';
-import dock from '@docknetwork/sdk';
-import { DockService } from './services/dock';
+import crypto from 'crypto';
 
-const keyring = new Keyring();
+const algorithm = 'aes-256-cbc';
 
-// const mnemonic = mnemonicGenerate(12);
-// 'wife village evoke error day record quick donor awful pass arctic arctic';
-const mnemonic =
-  "hole dog cross program hungry blue burst raccoon differ rookie pipe auction";
+const key = Buffer.from('776fe87eec8c9ba8417beda00b23cf22f5e134d9644d0a195cd9e0b7373760c1', 'hex');
+const iv = crypto.randomBytes(16);
 
-cryptoWaitReady().then(async () => {
-  await dock.init({
-    address: 'wss://knox-1.dock.io',
-  });
+function encrypt(text) {
+  let cipher = crypto.createCipheriv('aes-256-cbc', Buffer.from(key), iv);
+  let encrypted = cipher.update(text);
+  encrypted = Buffer.concat([encrypted, cipher.final()]);
+  return {iv: iv.toString('hex'), encryptedData: encrypted.toString('hex')};
+}
 
-  const testAccount = keyring.addFromUri(`${mnemonic}`);
+function decrypt(text) {
+  let iv = Buffer.from(text.iv, 'hex');
+  let encryptedText = Buffer.from(text.encryptedData, 'hex');
+  let decipher = crypto.createDecipheriv('aes-256-cbc', Buffer.from(key), iv);
+  let decrypted = decipher.update(encryptedText);
+  decrypted = Buffer.concat([decrypted, decipher.final()]);
+  return decrypted.toString();
+}
 
-  window.testAccount = testAccount;
-  window.dock = dock;
 
-  dock.setAccount(testAccount);
+function main() {
+    // const data = encrypt('test');
+    // const decrypted = decrypt(data);
+}
 
-  // const data = dock.api.tx.balances.transfer('37GfhtNUJk1aJhXuGxNJsAGenteDBX3DTVAvuBZm49Kqc9wA', '10000');
-  // const fee = await data.paymentInfo(testAccount);
+main();
+// import Keyring from '@polkadot/keyring';
```

Comment:
is there a plan to uncomment this code?

Reply:
I think we can remove this file, I've been using for some testing.